lab-teddy/README.md

```diff
@@ -0,0 +1,17 @@
+# Description:
```

Comment:
Good read me so far. In general you'll want to also explain the individual routes.